subsys/sdfw_services/services/device_info/device_info_service.cddl

```diff
@@ -0,0 +1,50 @@
+;
+; Copyright (c) 2024 Nordic Semiconductor ASA
+;
+; SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+;
+
+; struct typedefs
+; .. include_startingpoint_device_info_cddl_rst
+; Device Info service request payload.
+device_info_service_read_req = (
+      2,
+      target: uint, ; 0 - uuid, 1 - hw_version
+)
+
+device_info_service_write_req = (
```

Comment:
I assume that these are written in production already before any SD FW is even programmed?

Reply:
no it is not the case, this is a data model for the format of data to be exchanged between secure domain server and clients. nothing is written in production. 
this file dictates what is generated in the folder zcbor_generated.